mono/mini/mini-arm64.h

```diff
@@ -164,7 +164,7 @@ typedef struct {
 
 #endif
 
-#if defined(TARGET_APPLETVOS) || defined(TARGET_IOS)
+#if defined(TARGET_IOS) || defined(TARGET_APPLETV) || defined(TARGET_OSX)
```

Comment:
Change from `TARGET_APPLETVOS` to `TARGET_APPLETV` intentional here?

Reply:
It is not. Thanks for catching it. I'll fix it.

Reply:
Fixed in 4b04f88f04b9.